src/app.module.ts

```diff
@@ -7,6 +7,7 @@ import { RecipeModule } from './recipe/recipe.module';
 import { PrismaModule } from './prisma/prisma.module';
 import { RedisCacheModule } from './cache/redis-cache.module';
 import { WebSocketEventModule } from './websocket/websocket-event.module';
+import { WebhookModule } from './webhook/webhook.module';
```

Comment:
the folder should also be in plural - "webhooks". I'd leave it for some other PR as there might be a lot of changes in the code.